var/spack/repos/builtin/packages/amdblis/package.py

```diff
@@ -17,9 +17,22 @@ class Amdblis(BlisBase):
 
     _name = 'amdblis'
     homepage = "https://developer.amd.com/amd-aocl/blas-library/"
-    url = "https://github.com/amd/blis/archive/2.2.tar.gz"
+    url = "https://github.com/amd/blis/archive/3.0.tar.gz"
     git = "https://github.com/amd/blis.git"
 
     maintainers = ['amd-toolchain-support']
 
+    version('3.0', sha256='ac848c040cd6c3550fe49148dbdf109216cad72d3235763ee7ee8134e1528517')
     version('2.2', sha256='e1feb60ac919cf6d233c43c424f6a8a11eab2c62c2c6e3f2652c15ee9063c0c9')
+
+    def configure_args(self):
+        spec = self.spec
+        args = super(Amdblis, self).configure_args()
+
+        if spec.satisfies('@3.0%aocc'):
```

Comment:
```suggestion
        if spec.satisfies('@3.0 %aocc'):
```

Reply:
The above assumes the code in the body of the `if` statement applies to versions >= `3.0`.

Reply:
These flags are specific to amdblis 3.0 version. Next version of amdblis don't need below flags. 

 if spec.satisfies('@3.0 %aocc'):
            """ To enabled Fortran to C calling convention for
            complex types when compiling with aocc flang"""
            args.append('CFLAGS={0}'.format("-DAOCL_F2C"))
            args.append('CXXFLAGS={0}'.format("-DAOCL_F2C"))